### PR TITLE
Support Repeat1 formats for one-or-more

### PR DESCRIPTION
**Pull request by mikeday**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1417,8 +1441,8 @@ mod tests {
         assert!(Decoder::compile(&f, &Next::Empty).is_err());
     }
 
-    // This test takes over a minute to run
-    //#[test]
+    #[test]
+    #[ignore] // This test takes over a minute to run
```
Comment (adrianwong) [inline src/main.rs:1445]: Very minor (feel free to ignore):

```suggestion
    #[ignore = "this test takes over a minute to run"]
```

Comment (brendanzab) [reply src/main.rs:1445]: Oooh I didn't know about that!

**Review by brendanzab — APPROVED** [review]
Looks good (I think Adrian’s suggestion is good too)

File: src/main.rs (hunk 1 above)
Comment (mikeday) [reply src/main.rs:1445]: Ooh nice metadata!

**Review by brendanzab — APPROVED** [review]

**Pull request merged**